### PR TITLE
Update model name sanitizers according to the latest spec

### PR DESCRIPTION
**Pull request by pedrodanielcsantos**
Latest commit: https://github.com/Doist/doist-documentation/commit/bba1d5f0392d49b7b744708251c12a812d9cd3fb

For reference, here's the handbook link: https://handbook.doist.com/Product/Todoist/Name-sanitizers.md

### CODE REVIEW

File: src/main/java/com/todoist/pojo/Sanitizers.kt [hunk 1]
```diff
@@ -3,10 +3,9 @@ package com.todoist.pojo
 import java.util.regex.Pattern
 
 object Sanitizers {
-    const val PROJECT_NAME_INVALID_CHARACTERS = "<>\"=#+"
+    const val PROJECT_NAME_INVALID_CHARACTERS = "[#\"]+"
```
Comment (panpeter) [inline src/main/java/com/todoist/pojo/Sanitizers.kt:6]: This is now a regex but those should be only characters, no?

Comment (pedrodanielcsantos) [reply src/main/java/com/todoist/pojo/Sanitizers.kt]: Oh, of course! It was an oversight from my part when copy-pasting from the spec. 🤦‍♂️

**Comment by pedrodanielcsantos** [comment]
@panpeter Thank you very much for the review! I have amended the commit and rebased the branch with the latest changes in `master`. Please let me know if it looks good now! 💪

**Comment by pedrodanielcsantos** [comment]
Thanks a lot for the review @panpeter ! Merging this now!

**Pull request merged**